src/assets/js/add-article.js

```diff
@@ -0,0 +1,41 @@
+document.addEventListener("DOMContentLoaded", function() {
+    // Object containing arrays for each tag category
+    var tagCategories = {

```

Comment:
This part, we will not be able to access it, another thing to note, our tags don't have categories.
I know these are a little tough to speak about backend, but I am here to tell you these things won't be forgiven by companies.